builtin/logical/pki/path_tidy.go

```diff
@@ -0,0 +1,169 @@
+package pki
+
+import (
+	"crypto/x509"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathTidy(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "tidy",
+		Fields: map[string]*framework.FieldSchema{
+			"tidy_cert_store": &framework.FieldSchema{
+				Type: framework.TypeBool,
+				Description: `Set to true to enable tidying up
+the certificate store`,
+				Default: false,
+			},
+
+			"tidy_revocation_list": &framework.FieldSchema{
+				Type: framework.TypeBool,
+				Description: `Set to true to enable tidying up
+the revocation list`,
+				Default: false,
+			},
+
+			"safety_buffer": &framework.FieldSchema{
+				Type: framework.TypeDurationSecond,
+				Description: `The amount of extra time that must have passed
+beyond certificate expiration before it is removed
+from the backend storage and/or revocation list.
+Defaults to 72 hours.`,
+				Default: 259200, //72h, but TypeDurationSecond currently requires defaults to be int
+			},
+		},
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.UpdateOperation: b.pathTidyWrite,
+		},
+
+		HelpSynopsis:    pathTidyHelpSyn,
+		HelpDescription: pathTidyHelpDesc,
+	}
+}
+
+func (b *backend) pathTidyWrite(
+	req *logical.Request, d *framework.FieldData) (*logical.Response, error) {
+	safety_buffer := d.Get("safety_buffer").(int)
```

Comment:
The API docs do mention the data type; I'll fix up path-help. TypeDurationSecond *can* take an integer, but can also take a string -- it always outputs an integer though.

Not accepting it as the default is a bug, but not a user-facing one.